extensions/amp-3q-player/validator-amp-3q-player.protoascii

```diff
@@ -24,19 +24,37 @@ tags: {  # amp-3q-player
   }
   attr_lists: "common-extension-attrs"
 }
-tags: {  # <amp-3q-player>
-  html_format: AMP
-  tag_name: "AMP-3Q-PLAYER"
-  requires_extension: "amp-3q-player"
-  attrs: {
-    name: "autoplay"
-    value: ""
-  }
+
+attr_lists: { # <amp-3q-player> attributes that are common to both AMP & A4A.
```

Comment:
This has only one attribute, lets make that explicit within each `html_format` instead of creating a `attr_lists`.